babel.config.js

```diff
@@ -3,12 +3,12 @@ module.exports = function (api) {
   return {
     presets: ["babel-preset-expo"],
     plugins: [
-      "react-native-paper/babel",
       ["@babel/plugin-proposal-private-methods", { loose: true }],
       [
         "module-resolver",
         {
           alias: {
+            "@": "./src",
```

Comment:
fairly certain this is going to bust editor auto import. i think we should be getting first class support for aliases soon so I would hold off on this for now